tutorials/get-started-canonical-kubernetes/get-started-canonical-kubernetes.md

```diff
@@ -0,0 +1,187 @@
+---
+ 
+id: get-started-canonical-kubernetes
+summary: Learn how to operate a production-ready Kubernetes cluster. Kubernetes is a great open-source orchestration system for cloud native infrastructure. 
+categories: jaas 
+tags: jaas, juju, kubernetes, beginner, container, docker, cloud
+difficulty: 2
+status: Published
+Published: 2017-06-01
+ 
+---
+ 
+ 
+# Get started with Canonical Kubernetes
+ 
+ 
+## Overview
+Duration: 1:00
+ 
+### About the Canonical Distribution of Kubernetes
```

Comment:
The title has `Canonical Kubernetes` and here you refer to `Canonical Distribution of Kubernetes`. Which is it? We should be consistent.